wdl-lint/tests/lints/command-mixed-line-cont/source.wdl

```diff
@@ -3,19 +3,27 @@
 version 1.1
 
 task test1 {
```

Comment:
This is an interesting question that is probably out-of-scope. Long-term having these tests populated with additional fields that are irrelevant to the lint under test might complicate maintenance.

Reply:
I had that thought while I was manually changing all these tests. Hopefully next week we will have working lint directives that we can use to simplify these test cases. I would have preferred to add `#@ expect: MissingMetas,MissingOutput` to all these.

Reply:
I suppose the number of rules that are applied to every document will be small. Most are "if `x` exists, check `x`" so those will skip.